frontend/src/components/SideNavigationTeamOptions.vue

```diff
@@ -133,13 +133,15 @@ export default {
                 this.routes.admin.splice(1, 0, {
                     label: 'Billing',
                     to: '/billing',
+                    tag: 'team-billing',
```

Comment:
Not sure I'm following what these `tag` properties are bringing to the functionality?

Reply:
They're used for the `data-nav` tag, all the other items in the nav have it set further up in this file.  

I needed to navigate to the devices screen, and then added billing just for consistency.